package.json

```diff
@@ -4,7 +4,8 @@
   "description": "An open source project to practice client-server intergration",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "lint": "eslint 'src/**/*.{js,jsx}' --config ./.eslintrc.json",
```

Comment:
I think we are not going to have files with `jsx`  in the background, could you take care of remove them from here, we don't need to tell the linter we want to check that extension. Does that make sense?

Reply:
Did we solve this issue?

Reply:
done